app/models/Notifications.scala

```diff
@@ -156,6 +156,14 @@ object NotificationUserPreference {
       client.set(s"""NotificationUserPreference:${ConferenceDescriptor.current().eventCode}:${webUserId}""", json)
   }
 
+  def loadAll(webUserIds: List[String]): Map[String, NotificationUserPreference] = Redis.pool.withClient { implicit client =>
+    val notifUserPrefsKeys = webUserIds.map(webUserId => s"""NotificationUserPreference:${ConferenceDescriptor.current().eventCode}:${webUserId}""")
+    val notifPrefEntries = for ((webUserId, notificationPrefJSON) <- (webUserIds zip client.mget(notifUserPrefsKeys:_*)))
+      yield webUserId -> Json.parse(notificationPrefJSON).as[NotificationUserPreference]
+
+    notifPrefEntries.toMap
+  }
+
```

Comment:
Very very crappy fix to have it working 

```suggestion
  def loadAll(webUserIds: List[String]): Map[String, NotificationUserPreference] = Redis.pool.withClient { implicit client =>
    val notifUserPrefsKeys = webUserIds.map(webUserId => s"""NotificationUserPreference:${ConferenceDescriptor.current().eventCode}:${webUserId}""")
    if(notifUserPrefsKeys.isEmpty) return Map.empty[String, NotificationUserPreference]
    val notifPrefEntries = for ((webUserId, notificationPrefJSON) <- (webUserIds zip client.mget(notifUserPrefsKeys:_*)))
      yield webUserId -> Option(notificationPrefJSON).map(Json.parse(_).as[NotificationUserPreference]).getOrElse(DEFAULT_FALLBACK_PREFERENCES)

    notifPrefEntries.toMap
  }

```
if(notifUserPrefsKeys.isEmpty) return Map.empty[String, NotificationUserPreference]
```

fixes 

```
[JedisDataException: ERR wrong number of arguments for 'mget' command]
```

and

```
Option(notificationPrefJSON).map(Json.parse(_).as[NotificationUserPreference]).getOrElse(DEFAULT_FALLBACK_PREFERENCES)
```

A NPE on `notificationPrefJSON`

Then it looks to be ok

(Screenshots sent in private to not disclose my votes :-) )